x-pack/plugins/transform/public/app/sections/create_transform/components/step_details/step_details_form.tsx

```diff
@@ -500,6 +539,86 @@ export const StepDetailsForm: FC<Props> = React.memo(
               </EuiFormRow>
             </Fragment>
           )}
+
+          <EuiSpacer size="l" />
+
+          <EuiAccordion
+            data-test-subj="transformWizardAccordionAdvancedSettings"
+            id="transformWizardAccordionAdvancedSettings"
+            buttonContent={i18n.translate(
+              'xpack.transform.stepDetailsForm.advancedSettingsAccordionButtonContent',
+              {
+                defaultMessage: 'Advanced settings',
+              }
+            )}
+            paddingSize="s"
+          >
+            <EuiFormRow
+              label={i18n.translate('xpack.transform.stepDetailsForm.frequencyLabel', {
+                defaultMessage: 'Frequency',
+              })}
+              isInvalid={!isTransformFrequencyValid}
+              error={
+                !isTransformFrequencyValid && [
+                  i18n.translate('xpack.transform.stepDetailsForm.frequencyError', {
+                    defaultMessage: 'Invalid frequency format',
+                  }),
+                ]
+              }
+              helpText={i18n.translate('xpack.transform.stepDetailsForm.frequencyHelpText', {
+                defaultMessage:
+                  'The interval between checks for changes in the source indices when the transform is running continuously. Also determines the retry interval in the event of transient failures while the transform is searching or indexing. The minimum value is 1s and the maximum is 1h.',
+              })}
+            >
+              <EuiFieldText
+                placeholder="frequency"
```

Comment:
Can this have the same behavior as the edit form, where the placeholder text is much more user-friendly:

![image](https://user-images.githubusercontent.com/7405507/93222017-57346d80-f766-11ea-89d0-1c310b6be2ac.png)

Reply:
Checked latest edit and placeholder behavior in the Create wizard looks good.